spec/Auth.spec.js

```diff
@@ -90,6 +90,23 @@ describe('Auth', () => {
         });
       });
     });
+
+    it('should load all roles', async () => {
+      const user = new Parse.User();
+      user.id = currentUserId;
+      const roles = [];
+      for(let i = 0; i < 200;i++){
+        const acl = new Parse.ACL();
+        const role = new Parse.Role("roleloadtest" + i, acl);
+        role.getUsers().add([user]);
+        roles.push(role.save())
+      }
+      const savedRoles = await Promise.all(roles);
+      //Additionnal length check
+      expect(savedRoles).to.have.lengthOf(200);
+      const cloudRoles = auth.getUserRoles();
```

Comment:
Missing `await` here make sure to run `npm run test` before you commit. You may also specify a file 
`npm run test ./spec/Auth.spec.js`

Reply:
Indeed, I had a problem with my test environment

Reply:
Do you still have an issue?

Reply:
Yes i need help, my env with `npm run test ./spec/Auth.spec.js` create a `./lib` for tests, but if make somes changes in `./src/Auth.js` the cache is not actualised on `./lib/Auth.js`, i forget something ?

Reply:
https://github.com/parse-community/parse-server/blob/master/CONTRIBUTING.md

You should run watch